packages/react-components/react-color-picker-preview/library/etc/react-color-picker-preview.api.md

```diff
@@ -31,17 +31,21 @@ export type AlphaSliderSlots = ColorSliderSlots;
 export type AlphaSliderState = ComponentState<AlphaSliderSlots> & Pick<AlphaSliderProps, 'vertical'> & Omit<ColorSliderState, keyof ColorSliderSlots | 'components'>;
 
 // @public
-export const ColorArea: ForwardRefComponent<ColorAreaProps>;
+export const ColorArea: React_2.FC<ColorAreaProps>;
 
 // @public (undocumented)
 export const colorAreaClassNames: SlotClassNames<ColorAreaSlots>;
 
 // @public
-export type ColorAreaProps = ComponentProps<ColorAreaSlots> & {};
+export type ColorAreaProps = ComponentProps<ColorAreaSlots> & {
+    color?: string;
+    onChange?: EventHandler<ColorAreaOnColorChangeData>;
+};
 
 // @public (undocumented)
 export type ColorAreaSlots = {
-    root: Slot<'div'>;
+    root: NonNullable<Slot<'div'>>;
+    thumb?: NonNullable<Slot<'div'>>;
```

Comment:
Is it necessary for it to be mandatory if it's non-nullable, or can it be made optional and nullable?

Reply:
Unfortunately, it's necessary. It doesn't look logical, but it's a fix for error `Property 'thumb' is missing in type ...` There are other ways to fix it, like in this PR: #32323